### DISCUSSION TITLE
STNG - Lcars (portrait) [3.5"]

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#285] [Themes]
Got bored. Here is a simple LCARS theme, I have provided the needed font and PSD file should you want to edit the colors and layout.
The font needs to go into the {program root}/res/font folder for this theme to work. The zip file is laid out to be copied and pasted, and the paths should line up. **Update** - Didn't like how the words at the top looked, so made an update. Shrunk the degree and made the value larger to match the percentages. Added rounded corners. Changed some colors. This should be the final version of this theme.

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/35175147/4544a7e6-fc01-4d37-9c6f-6d4ed17e0a0c)

**Download**
[stng-lcars.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12030647/stng-lcars.zip)

**Comment by xfridgewizardx** [comment]
Can you make this in Landscape Mode too?
